src/test/regress/expected/partial_table.out

```diff
@@ -317,33 +317,31 @@ begin;
 	union all select * from t2 c join t2 d using(c2) ;
```

Comment:
`never executed` for this query is still here. Let's avoid touching variable part of plan (costs etc.) to decrease patch footprint. If this backport is ignored by upstream, we'll increase conflict area without explicit benefits.

As far I understand, all plans below changed only because prefetch for inner plan tree was implemented somedays and not depend on changes from this patch. Am I right?

Reply:
partial_table changes discarded